_infra/helm/locust/values.yaml

```diff
@@ -19,11 +19,13 @@ locust:
       security_user_name: admin
       security_user_password: secret
       test_respondent_password: password
-      test_respondents: 10
+      test_respondents: 1
       GOOGLE_CLOUD_PROJECT: "ras-rm-performance-20220908"
       GCS_BUCKET_NAME: "ras-rm-performance-20220908-locust"
       CSRF_ENABLED: true
       requests_file: requests.json
+      user_wait_time_min_seconds: 2
+      user_wait_time_max_seconds: 3
```

Comment:
Do we want to extend the [existing concourse helm task](https://github.com/ONSdigital/ras-rm-concourse/blob/main/tasks/helm/deploy/locust.yaml#L45-L48) to make use of this and allow us to override these? If so, we also need to change the original locust.py to apply the wait between GET/POST requests (see this branche's `locust_download_upload.py`)

Reply:
It makes sense to update them. I've updated them.